c2corg_ui/static/js/map/map.js

```diff
@@ -451,7 +451,7 @@ app.MapController.prototype.createPointStyle_ = function(feature, resolution) {
 
```

Comment:
Why is your PR including stuff related to map icons? I think you mixed different jobs?